scripts/backup-db.sh

```
@@ -61,13 +61,39 @@ fi
+
+if [ $? -eq 0 ]; then
+    log_info "Checksums generated successfully: $CHECKSUM_FILE"
+else
+    log_error "Failed to generate checksums"
+    exit 1
+fi
+
 # Upload to cloud storage
 case "$STORAGE_TYPE" in
     s3)
         upload_to_s3
```

Shellcheck (0.11.0)

[error] This function is only defined later. Move the definition up.

(SC2218)